scripts/build_palettes.ts

```diff
@@ -43,5 +44,26 @@ Promise.all(
       join(ROOT, `sip/${fname}.palette`),
       generateSip(fname, colors),
     );
+
+    const clrJson = join(ROOT, `clr/${fname}.json`);
+    await Deno.writeTextFile(
+      clrJson,
+      generateClrJson(fname, colors),
+    );
+    const cmd = new Deno.Command("swift", {
+      args: [
+        join(import.meta.dirname!, "./builders/palettes/json-to-clr.swift"),
+        clrJson,
+        join(ROOT, `clr/Catppuccin ${name}.clr`),
+      ],
+    });
+    const { code, stderr, stdout } = await cmd.output();
+    const td = new TextDecoder();
+    if (code === 0) {
+      console.log(td.decode(stdout).trim());
+    } else {
+      throw new Error(td.decode(stderr));
+    }
```

Comment:
This is pretty awful but it doesn't look like there is a better way besides pulling in another dep.